gateway_code/autotest/autotest.py

```diff
@@ -275,7 +275,7 @@ def auto_tests(self, channel=None, blink=False, flash=False, gps=False):
 
             # switch to DC and configure open node
             self._setup_open_node_connection()
-            time.sleep(1)
+            time.sleep(3)
```

Comment:
Is it possible to only use 3 when the autotests are run on an RPI2 gateway (or if the board type is Zigduino) ?
`board_type` is available with BoardConfig:
https://github.com/iot-lab/iot-lab-gateway/blob/2b2d86bd504d5a53a8e46d2f45c962712d0575e1/gateway_code/board_config.py#L32

Maybe we could add `gw_type` (but this needs to be done in the install_lib)